purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -451,6 +451,9 @@ data class PaywallData(
         @SerialName("tier_name")
         @Serializable(with = EmptyStringToNullSerializer::class)
         val tierName: String? = null,
+
+        @SerialName("offer_overrides")
+        val offerOverrides: Map<String, OfferOverride> = emptyMap(),
```

Comment:
Since these seem to be mostly overriding other existing values I was wondering whether it would make sense to just have the backend override these values in the existing fields instead of having the SDK process these overrides, so we have less logic on the SDKs (This does not apply to the badge which is a new field but could be added with the rest of localizations)... But maybe I'm missing something/don't want to add this logic to the backend, so I'm ok with this!

Reply:
@tonidero The backend right now only sends a single name and details to use across every package 😬 We really should have designed it this way from the start to make it more flexible but sadly did not 🤷‍♂️

Reply:
Ohhhh ok makes sense yeah. Thanks for explaining!